Paralayout/UIView+Distribution.swift

```diff
@@ -482,16 +482,16 @@ extension UIView : ViewDistributionSpecifying {
     /// horizontal positioning).
     /// `applySubviewDistribution([ 1.flexible, icon, 2.flexible ], orthogonalOffset: nil)`
     ///
+    /// - precondition: All views in the `distribution` must be subviews of the receiver.
+    /// - precondition: The `distribution` must not include any given view more than once.
```

Comment:
Maybe, we could translate these to actual `preconditionFailure`?

Reply:
These preconditions are checked in `ViewDistributionItem.items(impliedIn:axis:superview:)`. They result in a `fatalError`, rather than a `preconditionFailure`, which effectively the same but slightly better IMO (since it triggers even in `-Ounchecked` builds).